### ISSUE TITLE
addObject() not working

### ISSUE THREAD

**Issue by migbash** [typpo/spacekit#38]
I have an spaceObject that I store in an array[ ] of spaceObjects, which I use the remove from the simulation upon a button click, an then make the spaceObjects reappear on another button click using the array of spaceObjects and it does not work at all :/

**Comment by typpo** [comment]
hmm, is there any chance you can provide a jsfiddle as an example?

**Comment by migbash** [comment]
Yes, so when I try to remove some SpaceObjects() from the Simulation, and then add the same SpaceObjects back in, they do not work. I got them to work by re-creating the createObject() all over again, which I used as a quick hack.

![image](https://user-images.githubusercontent.com/20924663/103055980-95c29b00-4593-11eb-8abc-bcf01e872f36.png)
